app/src/main/AndroidManifest.xml

```diff
@@ -183,7 +183,7 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".reading.activity.ReadingActivity"
```

Comment:
Not sure why we need this.

Reply:
I have removed the unnecessary data from Reading Mode and moved the ReadingActivity file, hence the change in AndroidManifest.xml